src/keybindings.h

```diff
@@ -275,6 +275,9 @@ enum GeanyKeyBindingID
 	GEANY_KEYS_EDITOR_DELETELINETOBEGINNING,	/**< Keybinding. */
 	GEANY_KEYS_DOCUMENT_STRIPTRAILINGSPACES,	/**< Keybinding.
 												 * @since 1.34 (API 238) */
+	GEANY_KEYS_FORMAT_SENDTOCMD10,				/**< Keybinding. */
+	GEANY_KEYS_FORMAT_SENDTOCMD11,				/**< Keybinding. */
+	GEANY_KEYS_FORMAT_SENDTOCMD12,				/**< Keybinding. */
```

Comment:
You have added these at the end, good, but they are in the plugin API but you havn't increased the API number, and then you can add it to the comments like the previous line.